src/gpu/vulkan/SDL_gpu_vulkan.c

```diff
@@ -213,10 +213,11 @@ static VkFormat RefreshToVK_SurfaceFormat[] =
     VK_FORMAT_R16_UINT,					/* R16_UINT */
     VK_FORMAT_R16G16_UINT,				/* R16G16_UINT */
     VK_FORMAT_R16G16B16A16_UINT,		/* R16G16B16A16_UINT */
-    VK_FORMAT_D16_UNORM,				/* D16_UNORM */
-    VK_FORMAT_D32_SFLOAT,				/* D32_SFLOAT */
-    VK_FORMAT_D16_UNORM_S8_UINT,		/* D16_UNORM_S8_UINT */
-    VK_FORMAT_D32_SFLOAT_S8_UINT		/* D32_SFLOAT_S8_UINT */
+	VK_FORMAT_D16_UNORM,				/* D16_UNORM */
```

Comment:
inconsistent indentation (in case the sdl people care about that)

Reply:
Thanks, will address in the next push. That said, the formatting is generally a mess in these files. I think the plan is to clean everything up formally in one big pass once the dust has settled a bit.